GDEdit Web/GDE.Web/Data/GlobalVariables.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GDE.Web.Data
+{
+    public static class GlobalVariables
+    {
+        private static LinkItems currentSection = LinkItems.Home;
+        
+        public static LinkItems CurrentSection
+        {
+            get => currentSection;
+            set
+            {
+                if (Links.ContainsKey(value))
+                    currentSection = value;
+                else
+                    throw new Exception($"{value} is not inside the Links Dictionary");
+            }
+        }
+
+        public static Dictionary<LinkItems, string[]> Links = new Dictionary<LinkItems, string[]>
+        {
+            { 
+                LinkItems.Home,
+                new []
```

Comment:
I think I was clear enough about **not** including a space here, so that makes it `new[]`.

Reply:
oh sorry i didn't notice that